docs/DeveloperGuide.md

```diff
@@ -421,8 +421,6 @@ Team size: 5
 
 1. Currently, if any component (name, email, etc.) is too long, it will be hidden on the screen. We plan to implement
    appropriate mechanism to show the full component.
-2. Currently, names like "`John Doe`" and "`John  Doe`" (two spaces in the middle) are considered different. We plan
-   to have more sophisticated duplication detection mechanism.
 3. Currently, `undo` only supports undoing one command. We plan to support undoing multiple commands.
```

Comment:
renumbering?

Reply:
Doesn't matter, it will render correctly in the generated output anyway.